index.js

```diff
@@ -4704,6 +4712,7 @@
 //. [Either]:                   #either-type
 //. [Fantasy Land]:             v:fantasyland/fantasy-land
 //. [Foldable]:                 v:fantasyland/fantasy-land#foldable
+//. [Folktale]:                 https://folktale.origamitower.com/
```

Comment:
You may like to update the link at <https://github.com/folktale> to use HTTPS, @robotlolita. :)